src/perception/tag_detector/tag_detector.processing.cpp

```diff
@@ -122,7 +124,7 @@ namespace mrover {
                     // Publish tag to odom
                     std::string const& parentFrameId = mUseOdom ? mOdomFrameId : mMapFrameId;
                     SE3 tagInParent = SE3::fromTfTree(mTfBuffer, parentFrameId, immediateFrameId);
-                    SE3::pushToTfTree(mTfBroadcaster, "fiducial" + std::to_string(id), parentFrameId, tagInParent);
+                    SE3::pushToTfTree(mTfBroadcaster, "fiducial", parentFrameId, tagInParent);
```

Comment:
What happens if multiple tags are visible?

I would imagine we want to:
1) publish the closer one
2) publish all (would require making the message an array of strings)

Reply:
I actually think it probably doesn't matter. In the worst case, where you see two tags at the same time so your "target" tag starts oscillating between them, you will keep moving forward towards them and eventually you will either reach one or one of them will go out of your FOV.

As far as I can tell from the circ rules, the only time this would be a problem is while going to the first two waypoints, since they need to be done in order. So if we see another tag on the way to waypoints one or two and we start going towards it, that would be a problem and we would have to restart, but I think that's very unlikely.

btw does anyone remember what we did last year to solve this problem?